.circleci/config.yml

```diff
@@ -137,6 +146,20 @@ jobs:
           name: release
           command: bundle exec fastlane release
 
+  update-hybrid-common-versions:
+    description: "Creates a PR updating purchases-hybrid-common to latest release"
+    docker:
+      - image: cimg/ruby:3.1.2
+    steps:
+      - checkout
+      - revenuecat/install-gem-unix-dependencies:
+          cache-version: v1
+      - revenuecat/trust-github-key
+      - revenuecat/setup-git-credentials
+      - run:
+          name: Prepare next version
+          command: bundle exec fastlane update_hybrid_common version:<< pipeline.parameters.hybrid-common-version >> open_pr:true automatic_release:true
```

Comment:
I guess the `automatic_release` flag would depend on whether this job is run from the manual workflow or thorough the scheduled workflow right? So it should be a parameter of the job?

Reply:
hmm good point. It's the same in `purchases-flutter`, where phc triggers the `update-hybrid-common-versions` via API after a release is made. So I am going to add it as a parameter and phc will set it to true when triggering the workflow via API

Reply:
I am also going to change `hybrid-common-version` to `version`, because I always have to check `hybrid-common-version`